challenges/backend/src/app/services/CarOnSaleClient/classes/Calculator.ts

```diff
@@ -0,0 +1,37 @@
+import { injectable } from "inversify";
+import { AuctionItem } from "../dtos/auction/AuctionItem";
+
+@injectable()
+export class Calculator {
+    calculatePercentage(currentHighestBidValue: number, minimumRequiredAsk: number | null):number {
```

Comment:
The class method `calculatePercentage` must be marked either 'private', 'public', or 'protected'